kustomize/apps/centraldashboard/base/deployment.yaml

```diff
@@ -20,3 +20,7 @@ spec:
       serviceAccountName: centraldashboard
       imagePullSecrets:
       - name: k8scc01covidacr-registry-connection
+      resources:
```

Comment:
this will make it such that if it goes over the requests since there were no limits it may get throttled, we could just omit this or add a limit field to account for any resource spikes